packages/vscode-extension/src/qm/vsc_ui.ts

```diff
@@ -504,9 +504,6 @@ export class VsCodeUI implements UserInteraction {
 
           disposables.push(
             quickPick.onDidAccept(onDidAccept),
-            quickPick.onDidHide(() => {
```

Comment:
Is there any side effects when user press Esc in the quick pick?

Reply:
ESC can still cancel the quick pick. It invokes `onDidHide` as well.

Reply:
When it invokes `onDidHide`, will it trigger UserCancelError and send correct telemetry?

Reply:
Yes, I'll update the code. Otherwise, the telemetry will be lost.